tests/pkcli/build_data/1.out/srv/host/v9.radia.run/srv/vm_devbox_user_1/start

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -eou pipefail
+
+vm_devbox_first_start() {
+    if [[ -e Vagrantfile ]]; then
+        return
+    fi
+    curl 'https://radia.run' | vagrant_dev_vm_devbox=1 bash -s vagrant-sirepo-dev 'user_1.radia.run'
+}
+
+vm_devbox_set_forwarded_port() {
+    python <<'EOF'
+import re
+import subprocess
+
+_FILE = "Vagrantfile"
+_FORWARDED_PORT_PREAMBLE = 'config.vm.network "forwarded_port"'
+_FORWARDED_PORT_FULL = (
+    "{}, guest: 11110, host: 11110".format(
+        _FORWARDED_PORT_PREAMBLE
+    )
+)
+r = False
+with open(_FILE, "r+") as f:
+    c = f.read()
+    if _FORWARDED_PORT_PREAMBLE not in c:
+        c = re.sub(
+            r"^\s*(config\.vm\.hostname.*)$",
+            r"\1\n{}".format(_FORWARDED_PORT_FULL),
+            c,
+            flags=re.MULTILINE,
+        )
+        r = True
+    elif _FORWARDED_PORT_FULL not in c:
+        c = re.sub(
+            r"^\s*{}.*$".format(_FORWARDED_PORT_PREAMBLE),
+            _FORWARDED_PORT_FULL,
+            c,
+            flags=re.MULTILINE,
+        )
+        r = True
+    f.seek(0)
+    f.write(c)
+    f.truncate()
```

Comment:
Why go through this complexity? Just read file and write it again.

Reply:
a6e3aa2